src/marked.js

```diff
@@ -204,6 +251,9 @@ marked.walkTokens = function(tokens, callback) {
         break;
       }
       default: {
+        if (this.options.extensions?.walkableTokens?.[token.type]) { // Walk any extensions
+          marked.walkTokens(this.options.extensions.walkableTokens[token.type], callback);
+        }
```

Comment:
> Is walkableTokens supposed to be the list of tokens or the names of the properties for tokens?

Currently It's just a list of tokens, but I'm seeing now that you can't define a list of tokens in the extension since the tokens are not generated at that point. I will change it to a list of properties of the token.

> Can you add a test with walkable tokens?

I could use some help coming up with a walkTokens unit test. I never really use that feature and I'm not sure what good use cases/bugs we need to catch.

> If the walkable tokens are in the tokens property will they be walked twice?

Currently yes. I can fix that so it only walks `tokens` once.

Reply:
Alright, these are all fixed now.